unit-tests/live/intrinsics/test-motion-intrinsics.py

```diff
@@ -0,0 +1,33 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+# test:device each(D400*)
+
+import pyrealsense2 as rs
+from rspy import test
+
+
+device = test.find_first_device_or_exit()
+motion_sensor = device.first_motion_sensor()
+
+if motion_sensor:
```

Comment:
Please add`test.check( motion_sensor )`

Reply:
D435 has no motion sensor. It will fail test.

Reply:
In this case, here is your issue:
`# test:device each(D400*)`

you want 
`# test:device D455`
try it..

Reply:
Are we want test D457?

Reply:
Yes, also add D457

Reply:
Noted

Reply:
what about D435i?

Reply:
We dont have it in LibCI IMO